lab01/excercise-01/src/App.js

```diff
@@ -0,0 +1,78 @@
+import imgUser from './assets/img/imgUser.jpg';
+import './App.css';
+
+function App() {
+  return (
+    <>
+    <div className="header"><h1 className="title-header">Bài 01</h1></div>
+    <div className="container">
+      <div className="container-lab01">
+        <div className="img-left-wrap">
+          <img className="img-left" src={imgUser} alt="ảnh user" />
+        </div>
+
+        <div className="info-right-wrap">
+          <h3>Họ và tên: Lê Hoài Nam</h3>
+          <h3>Giới tính: nam</h3>
+          <h3>Địa chỉ:Tổ 4, Thôn 4, Xã Mê Pu, Huyện Đức Linh - Bình Thuận</h3>
+          <h3>Số điện thoại: 0363556909</h3>
+          <h3>Email: lehnam2991@gmail.com</h3>
+          <h3>Zalo: 0363556909</h3>
+        </div>
+      </div>
+
+      <div className="header"><h1 className="title-header">Bài 02</h1></div>
+      <div className="container-lab02">
+        <h1 className="title-lab02">Hello, I am Lê Hoài Nam</h1>
+        <h2 className="description-lab02">Now is 8:17:05 PM</h2>
```

Comment:
Chỗ "Now is 8:17:05 PM" viết component riêng và dùm javascript để viết hàm lấy ngày giờ hệ thống. Link tham khảo đây nhé https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_tolocaletimestring